src/logger.js

```diff
@@ -0,0 +1,23 @@
+const bunyan = require('bunyan')
+
+const LEVELS = [
+  "info", "fatal", "error", "warn", "info", "debug", "trace", 
+]
+
+class Logger {
+  constructor(name) {
+    this.instance = bunyan.createLogger({ name })
+  }
+
+  log (level, ...rest) {
+    if (LEVELS.includes(level)) {
+      this.instance[level]({
```

Comment:
For a general `version` field, fields can be added to every entry by defining them in the bunyan constructor: https://github.com/trentm/node-bunyan#constructor-api